api/src/main/java/org/openmrs/module/dhisreporting/Configurations.java

```diff
@@ -19,6 +19,10 @@ private Concept getConceptByGpCode(String globalPropertyCode) {
 		return conceptId != null ? Context.getConceptService().getConcept(conceptId) : null;
 	}
 
+	public Boolean getDxfToAdxSwitchConcept() {
+		return "true".equals(getConceptByGpCode(DHISReportingConstants.DXF_TO_ADX_SWITCH));
```

Comment:
thanks @BMartinos, this however still reads this dxfToAdxSwitch as a concept, it should read just the value set by the user and check if  it is true. an equivalent method is here: https://github.com/jembi/openmrs-module-dhisconnector/blob/master/api/src/main/java/org/openmrs/module/dhisconnector/Configurations.java#L11-L13